examples/word-count/README.md

```diff
@@ -0,0 +1,44 @@
+Word-Count (a.k.a. the "Hello, World!" for data analytics) counts the occurrences of each distinct word from a text file.
+
+This examples shows the *killer feature* in the PiCo API: **there is no data**!
+A PiCo application is described only in terms of *processing* (i.e., pipeline stages) -
+rather than processing *and* data.
+
+In `pico_wc.cpp`, the `wc` pipeline:
+
+1. reads an input file line by line, by a `ReadFromFile` stage
+2. tokenizes each line into words, by a `FlatMap` stage - a `FlatMap` maps an item (line) to multiple items (words)
+3. maps each word `w` to a key-value pair <`w`,`1`>, by a `Map` stage
```

Comment:
There is no Map stage. The transformation w -> <w,1> is performed by the FlatMap stage

Reply:
Fixed.